jpa/bin/pom.xml

```diff
@@ -157,6 +195,17 @@
                                     <destFileName>dbprocedures.jar</destFileName>
                                 </artifactItem>
                             </artifactItems>
+                            <!--
```

Comment:
do not leave commented out code in. Either add a note why it is commented out (and when one should uncomment this) or remove it

Reply:
removed commented code.